### PR TITLE
RUBY-1413 modified Change Stream Example 4

### PR DESCRIPTION
**Pull request by Neilshweky**
I couldn't get the code to run on my local machine but I used the same syntax as what was there before and just modified the variable names/pipeline object, so it should be fine.

### CODE REVIEW

File: spec/integration/change_stream_examples_spec.rb [hunk 1]
```diff
@@ -174,10 +174,12 @@
...
 
-        pipeline = [ {'$match' => { '$or' => [{ 'fullDocument.username' => 'alice' },
-                                              { 'operationType' => 'delete' }] } }]
-        cursor = inventory.watch(pipeline).to_enum
-        cursor.next
+        pipeline = [
+          { "$match" => { 'fullDocument.username' => 'alice' } },
+          { "$addFields" => { 'newField' => 'this is an added field!' } }
+        ];
+        changeStream = inventory.watch(pipeline).to_enum
```
Comment (p-mongo) [inline spec/integration/change_stream_examples_spec.rb:181]: Let's use `cursor` for consistency with other examples.

File: spec/integration/change_stream_examples_spec.rb [hunk 2]
```diff
@@ -174,10 +174,12 @@
...
+        changeStream.next
 
         # End Changestream Example 4
```
Comment (p-mongo) [inline spec/integration/change_stream_examples_spec.rb:184]: After this line please add an assertion that the document retrieved from the change stream has the expected fields (i.e. the `newField` is present). You can look at other examples to see how they assert after the example itself.

Comment (Neilshweky) [reply spec/integration/change_stream_examples_spec.rb]: Done!

File: spec/integration/change_stream_examples_spec.rb (hunk 1 above)
Comment (Neilshweky) [reply spec/integration/change_stream_examples_spec.rb]: Done!

**Pull request merged**